consensus/polybft/consensus_runtime.go

```diff
@@ -391,10 +303,12 @@ func (c *consensusRuntime) FSM() error {
 	}
 
 	if c.IsBridgeEnabled() {
-		err := c.populateFsmIfBridgeEnabled(ff, epoch, parent, isEndOfEpoch, isEndOfSprint)
+		commitment, err := c.stateSyncManager.Commitment()
```

Comment:
This means that you can register commitment in any block in an epoch, but be aware, if you do not have quorum (which is a valid case), you will return error here, which is not something we want to do in that case, right?

Another issue is, I don't see anywhere that you reset that commitment field in `stateSyncManager` after you successfully submit a commitment after you reach quorum. Basically, you can submit the same epoch more than once in epoch.

Reply:
Resolved comment in: https://github.com/0xPolygon/polygon-edge/pull/1067/commits/503224004886fb793d1d63d972144d5323a9e546